commands/FBPrintCommands.py

```diff
@@ -299,3 +300,27 @@ def run(self, arguments, options):
       object = fb.evaluateObjectExpression(objectToMessage)
       printCommand = 'po [{} valueForKeyPath:@"{}"]'.format(object, keypath)
       lldb.debugger.HandleCommand(printCommand)
+
+
+class FBPrintApplicationDocumentsPath(fb.FBCommand):
+  def name(self):
+    return 'pdocspath'
+
+  def description(self):
+    return "Print application's 'Documents' directory path."
+  
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-o', long='--open', arg='open', boolean=True, default=False, help='open in Finder'),
+    ]
+
+  def run(self, arguments, options):
+    if options.open:
+      path = fb.evaluateExpressionValue('(NSString *)[NSSearchPathForDirectoriesInDomains(9, 1, YES) lastObject]')
+      pathString = '{}'.format(path).split('"')[1]
+      print pathString
+      lldb.debugger.HandleCommand('script os.system("open {}")'.format(pathString))
```

Comment:
This can call `os.system()` directly, it's not necessary to call it via `HandleCommand()`.